drivers/net/can/usb/gs_usb.c

```diff
@@ -1117,6 +1193,10 @@ static struct gs_can *gs_make_candev(unsigned int channel,
 		kfree(bt_const_extended);
 	}
 
+	dev->can.termination_const = gs_termination;
+	dev->can.termination_const_cnt = ARRAY_SIZE(gs_termination);
+	dev->can.do_set_termination = gs_set_termination;
+
```

Comment:
We cannot do this unconditionally...Please set `GS_CAN_FEATURE_TERMINATION BIT` if your hardware supports it.

Reply:
Took me a minute to rationalize this request... yeah you're right.